reana_client/cli/utils.py

```diff
@@ -160,3 +160,62 @@ def handle_parse_result(self, ctx, opts, args):
 
         return super(NotRequiredIf, self).handle_parse_result(
             ctx, opts, args)
+
+
+def output_user_friendly_logs(workflow_logs, steps):
+    """Output workflow logs in an user-friendly manner.
+
+    :param workflow_logs: Dictionary representing the workflow logs as they
+        are returned from the REST API.
+    :param steps: List of steps to show logs for.
+    """
+    def _leading_mark(indent=1):
+        return '{}>'.format('='*indent)
+    # Colors matching the status colors of REANA-UI
+    status_to_color_mapping = {
+        'created': 'magenta',
+        'running': 'bright_blue',
+        'finished': 'green',
+        'failed': 'red',
+        'stopped': 'yellow',
+        'deleted': 'black',
+        'queued': 'bright_white',
+    }
+
+    # REANA Workflow Engine logs
+    if workflow_logs.get('workflow_logs', None):
+        click.secho('{} workflow engine logs'.format(_leading_mark()).upper())
+        click.echo(workflow_logs['workflow_logs'])
+
+    returned_step_names = \
+        set(workflow_logs['job_logs'][item]['job_name']
+            for item in workflow_logs['job_logs'].keys())
+    if steps:
+        missing_steps = set(steps).difference(returned_step_names)
+        if missing_steps:
+            click.echo(
+                click.style('The logs of step(s) {} were not found, '
+                            'check for spelling mistakes in the step '
+                            'names.'
+                            .format(','.join(missing_steps)),
+                            fg='red'))
+    # Job logs
+    if workflow_logs['job_logs']:
+        click.echo('\n')
+        click.secho('{} job logs'.format(_leading_mark()).upper())
+    for job_id, logs_info in workflow_logs['job_logs'].items():
+        if logs_info:
+            click.secho('{0} Step {1} (ID {2})'.format(
+                _leading_mark(indent=2), logs_info['job_name'], job_id),
+                fg=status_to_color_mapping[logs_info['status']], bold=True)
+            del(logs_info['job_name'])
+            for key, value in logs_info.items():
+                click.secho(
+                    '{mark} {key}: {value}'.format(
+                        mark=_leading_mark(indent=3),
+                        key=key.upper().replace('_', ' '),
```

Comment:
Hmm, still keeping `upper()`?  When we shall have a nice leading mark and display the leading "job name" labels in colour, it is perhaps not necessary?

We could also amend wording, e.g. `DOCKER_IMG` -> `==> Docker image:`, `CMS` -> `==> Command:`, as these are the only two abbreviated names within the set; elsewhere we use full words

(sorry cannot test easily right now different looks)

Reply:
Screenshot from Gruvbox terminal color scheme:

![2020-02-14-143842_532x535_scrot](https://user-images.githubusercontent.com/517546/74536127-bbc0ed00-4f37-11ea-915c-ae4b7b3dd96d.png)

Notes:

- `==> JOB LOGS` is not coloured, while it is the top node; could be at least _bold_ in default terminal colour

- `===> COMPUTE BACKEND` labels are not coloured; could be (1) using some neutral blueish colour (like '::' in `yay`), or (2) could in the same colour as the step (but the step name would be in _bold_ and this label in normal font); or (3) could be simply in the default terminal colour (as is the case now) but _bold_ to attract eye attention.

It would help for orientation inside large workflow files.

- For unnamed steps, I'm seeing:

```
==> Step  (ID 388ed71b-9130-4ac8-8124-bb454a384537)
```

Note two spaces; one can be deleted.

Reply:
Updated with all requested changes.

Regarding:
> ===> `COMPUTE BACKEND` labels are not coloured; could be (1) using some neutral blueish colour (like '::' in `yay`), or (2) could in the same colour as the step (but the step name would be in bold and this label in normal font); or (3) could be simply in the default terminal colour (as is the case now) but bold to attract eye attention.

Options (1) and (2) were too much, going for (3) as it is the least disturbing to the eyes but still conveying the message.

Regarding:
> For unnamed steps, I'm seeing:
==> Step  (ID 388ed71b-9130-4ac8-8124-bb454a384537)

I went for showing the name if available, otherwise show the ID.
![image](https://user-images.githubusercontent.com/9035606/74539679-c468f180-4f3e-11ea-8b4e-45299269c038.png)
![image](https://user-images.githubusercontent.com/9035606/74539789-ea8e9180-4f3e-11ea-9480-e5f42e6652fe.png)

Reply:
- `Job id in compute backend` -> `Job ID`
- `Logs output` -> `Logs` one word enough?  Or better `Output` one word enough? Both seem a bit redundant...  We could also use `Standard output` (and later `Standard error`) that might be the best if one day we can split them

Reply:
Updated, I've just used `Logs` as standard output seems a bit too much, and it will raise the question about standard error... let me know what you think :)